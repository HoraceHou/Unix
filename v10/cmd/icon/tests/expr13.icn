record array(a,b,c,d,e,f,g)

procedure dummy(u,v,x,y,z)
   suspend u | v
   return x
end

procedure main()
   &trace := 2
   dummy(1)
   dummy(1,.20,&null,[])
   &trace := -1
   dummy(1,2.0,&null,[])
   dummy()
end
