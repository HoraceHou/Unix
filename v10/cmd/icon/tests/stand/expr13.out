expr13.icn: 10	| dummy(1,&null,&null,&null,&null)
expr13.icn: 4	| dummy suspended 1
expr13.icn: 13	| dummy(1,2.0,&null,[],&null)
expr13.icn: 4	| dummy suspended 1
expr13.icn: 14	| dummy(&null,&null,&null,&null,&null)
expr13.icn: 4	| dummy suspended &null
expr13.icn: 15	main failed
